tests/sentry/backup/test_correctness.py

```diff
@@ -1,111 +1,40 @@
-from __future__ import annotations
-
-from difflib import unified_diff
 from pathlib import Path
-from typing import NamedTuple
 
 import pytest
 from click.testing import CliRunner
 from freezegun import freeze_time
 
 from sentry.db.postgres.roles import in_test_psql_role_override
-from sentry.runner.commands.backup import export, import_
+from sentry.runner.commands.backup import ComparatorFindings, export, import_, validate
 from sentry.testutils.factories import get_fixture_path
 from sentry.utils import json
-from sentry.utils.json import JSONData, JSONEncoder, better_default_encoder
-
-
-# TODO(team-ospo/#155): Figure out if we are going to use `pk` as part of the identifier, or some other kind of sequence number internal to the JSON export instead.
-class InstanceID(NamedTuple):
-    """Every entry in the generated backup JSON file should have a unique model+pk combination, which serves as its identifier."""
-
-    model: str
-    pk: int
-
-    def pretty(self) -> str:
-        return f"InstanceID(model: {self.model!r}, pk: {self.pk})"
-
-
-class ComparatorFinding(NamedTuple):
-    """Store all information about a single failed matching between expected and actual output."""
-
-    name: str
-    on: InstanceID
-    reason: str = ""
-
-    def pretty(self) -> str:
-        return f"Finding(\n\tname: {self.name!r},\n\ton: {self.on.pretty()},\n\treason: {self.reason}\n)"
-
-
-class ComparatorFindings:
-    """A wrapper type for a list of 'ComparatorFinding' which enables pretty-printing in asserts."""
 
-    def __init__(self, findings: list[ComparatorFinding]):
-        self.findings = findings
 
-    def append(self, finding: ComparatorFinding) -> None:
-        self.findings.append(finding)
+class ValidationError(Exception):
+    def __init__(self, info: ComparatorFindings):
+        super().__init__(info.pretty())
+        self.info = info
 
-    def pretty(self) -> str:
-        return "\n".join(f.pretty() for f in self.findings)
 
+def django_db_complete_reset(func=None):
```

Comment:
should this be moved to the `src/sentry/utils/pytest/fixtures.py`?

Reply:
Done; I've added this as a keyword param to `django_db_all` instead.